autoload/gopher/coverage.vim

```diff
@@ -94,7 +94,7 @@ endfun
 fun! gopher#coverage#stop() abort
   let s:visible = 0
   let s:coverage = {}
-  silent! au! gopher-coverage
+  silent! au! gopher.vim-coverage
```

Comment:
out of context.

is this typo?

Reply:
It's intentional as most things are prefixed with just "gopher" rather than "gopher.vim"; for example settings, mappings, etc.